### PR TITLE
Azure Communication Service - Phone Number Administration - Samples

### PR DESCRIPTION
**Pull request by ytyeason**
Add initial samples for TNM Python.

### CODE REVIEW

File: sdk/communication/azure-communication-administration/samples/phone_number_configuration_sample.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    PstnConfiguration
+)
+
+class CommunicationPhoneNumberConfigurationSamples(object):
+
+    def __init__(self):
+        self.connection_str = os.getenv('AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING')
+        self._phone_number_administration_client = PhoneNumberAdministrationClient.from_connection_string(
+            self.connection_str)
+        self.phonenumber_to_configure = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_PHONENUMBER_TO_CONFIGURE',
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-administration/samples/phone_number_configuration_sample.py:30]: let's remove "TNM_" we do should not use this abbreviation anymore

File: sdk/communication/azure-communication-administration/samples/phone_number_area_codes_sample.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# --------------------------------------------------------------------------
+
+"""
+FILE: phone_number_area_codes_sample.py
+DESCRIPTION:
+    These samples demonstrate area codes samples.
+
+    ///getting all area codes via a connection string, country code and phone plan id
+USAGE:
+    python phone_number_area_codes_sample.py
```
Comment (xiangyan99) [inline sdk/communication/azure-communication-administration/samples/phone_number_area_codes_sample.py:16]: Let's list required environment variables here. 

You can take https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/metricsadvisor/azure-ai-metricsadvisor/samples/sample_data_feeds.py as a reference.

File: sdk/communication/azure-communication-administration/samples/phone_number_area_codes_sample_async.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            all_area_codes = await self._phone_number_administration_client.get_all_area_codes(
+                location_type="NotRequired",
+                country_code=self.country_code,
+                phone_plan_id=self.phone_plan_id_area_codes
+            )
+        print('all_area_codes:')
+        print(all_area_codes)
+
+async def main():
+    sample = CommunicationAreaCodesSamplesAsync()
```
Comment (xiangyan99) [inline sdk/communication/azure-communication-administration/samples/phone_number_area_codes_sample_async.py:42]: Please don't use classes here. Instead, just use methods.

We don't recommend user to have a CommunicationAreaCodesSamplesAsync to achieve what they want. :)

File: sdk/communication/azure-communication-administration/samples/phone_number_area_codes_sample.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: phone_number_area_codes_sample.py
+DESCRIPTION:
+    This sample demonstrates how to get all area codes via a connection string, country code and phone plan id.
+USAGE:
+    phone_number_area_codes_sample.py
```
Comment (xiangyan99) [inline sdk/communication/azure-communication-administration/samples/phone_number_area_codes_sample.py:14]: I think you want to have "python phone_number_area_codes_sample.py"?

Comment (ytyeason) [reply sdk/communication/azure-communication-administration/samples/phone_number_area_codes_sample.py:14]: yes, you're right. will fix it, thanks!

**Review by xiangyan99 — APPROVED** [review]

**Pull request merged**